yarn.lock

```diff
@@ -2532,16 +2421,7 @@
     npmlog "^4.1.2"
     write-file-atomic "^2.3.0"
 
-"@mdx-js/loader@^1.5.1":
```

Comment:
I only manually deduplicated `@mdx-js` packages. The rest is just a result of running `yarn` after that.